content/chapter/05.tex

```diff
@@ -1,3 +1,19 @@
 %!TEX root = ../../main.tex
 
-\chapter{Ausblick}
\ No newline at end of file
+\chapter{Ausblick}
+Aufgrund der technischen Mängel und Probleme der Sensoren, welche in dieser Studienarbeit verwendet wurden, war es nicht möglich, das erhoffte Ergebnis der Arbeit zu erzielen.
```

Comment:
du schreibst hier immer wieder in der Vergangenheitsform passt des so für den Ausblick?